pages/devs/reputers/reputers.mdx

```diff
@@ -1,150 +1,214 @@
+
+### `wallet` Sub-object
+
+1. `nodeRpc`: The [RPC URL](/devs/get-started/setup-wallet#rpc-url-and-chain-id) for the corresponding network the node will be deployed on
+
+#### Optional
+
+2. `addressKeyName`: The name you gave your wallet key when [setting up your wallet](/devs/get-started/setup-wallet)
+3. `addressRestoreMnemonic`: The mnemonic that was outputted when setting up a new key
+
+<Callout type="info">
+`addressKeyName` and `addressRestoreMnemonic` are optional parameters. If you did not previously generate keys, keys will be generated for you when [running the node](/devs/workers/deploy-worker/using-docker#generate-keys-and-export-variables).
+
+If you have existing keys that you wish to use, you will need to provide these variables.
+</Callout>
+
+### `reputer` Config
+
+1. `topicId`: The specific topic ID you created the reputer for.
+2. `SourceOfTruthEndpoint`: The endpoint exposed by your source of truth server to provide the truth data to the network.
+3. `Token`: The token for the specific topic you are verifying truth data for. This token should be included in the source of truth endpoint for retrieval.
+   - The `Token` variable is specific to the endpoint you expose in your `main.py` file. It is **not** related to any blockchain parameter and is only locally specific.
+4. `minStake`: The minimum stake required to participate as a reputer. This stake will be deducted from the reputer's wallet balance.
+
+<Callout>
+When placing your minimum stake, the system will verify the amount of funds you have already staked in the topic. If your staked amount is insufficient, it will automatically pull the necessary funds from your wallet to meet the required minimum.
+</Callout>
+
+<Callout type="warning">
+The `reputer` config is an array of sub-objects, each representing a different topic ID. This structure allows you to manage multiple topic IDs, each within its own sub-object.
+
+To deploy a reputer that provides inferences for multiple topics, you can duplicate the existing sub-object and add it to the `reputer` array. Update the `topicId`, `SourceOfTruthEndpoint`, `minStake` and `Token` fields with the appropriate values for each new topic:
+```json
+"worker": [
+      {
+          "topicId": 1,
+          "reputerEntrypointName": "api-worker-reputer",
+          "loopSeconds": 30,
+          "minStake": 100000,
+          "parameters": {
+              "SourceOfTruthEndpoint": "http://source:8888/truth/{Token}/{BlockHeight}",
+              "Token": "ethereum"
+          }
+      },
+      // reputer providing ground truth for topic ID 2
+      {
+          "topicId": 2,
+          "reputerEntrypointName": "api-worker-reputer",
+          "loopSeconds": 30,
+          "minStake": 100000,
+          "parameters": {
+              "SourceOfTruthEndpoint": "http://source:8888/truth/{Token}/{BlockHeight}",
+              "Token": "ethereum"
+          }
+      }
+    ],
 ```
-docker compose up
-```
+</Callout>
```

Comment:
Currently with this config , because we don't tell the user to "remove the reputer part", it will bring up both a worker and a reputer. But this tutorial is just about worker, right?

In order to just run a worker as the title says, pls tell the user to remove the reputer block, and/or advise them that if not removed the reputer part, it will also run reputer.

Reply:
Makes sense.